ha/core/event_analyzer/parser/parser.py

```diff
@@ -71,3 +80,44 @@ def parse_event(self, msg: str) -> HealthEvent:
         except Exception as e:
             raise EventAnalyzerError(f"Failed to parse alert. Message: {msg}, Error: {e}")
 
+class IEMParser(Parser):
+    """
+    Parser for IEMs.
+    """
+
+    def parse_event(self, msg: str) -> HealthEvent:
+        """
+        Parse event.
+        Args:
+            msg (str): Msg
+        """
+        try:
+            iem_alert = json.loads(msg)
+
+            # Parse hostname and convert to node id
+            iem_description = iem_alert['sensor_response_type']['info']['description']
```

Comment:
import Template 
can be used to resolve value

https://github.com/Seagate/cortx-ha/blob/cortx-1.0/ha/alert/pcs/alert_generator.py#L121

Reply:
Yes, I am using Template for substituting the value in the test file. But here we need to read the already substituted value. Do you think the Template will help here too?